cmd/config/constants.go

```diff
@@ -33,6 +33,10 @@ const (
 	EnvPublicIPs    = "MINIO_PUBLIC_IPS"
 	EnvEndpoints    = "MINIO_ENDPOINTS"
 
+	// API sub-system
+	EnvAPIConnMax      = "MINIO_API_CONN_MAX"
+	EnvAPIConnDeadline = "MINIO_API_CONN_DEADLINE"
```

Comment:
This is already decided @klauspost keeping it as _CONN for now because it says its API connections, requests is also fine will check with @abperiasamy